connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -670,10 +711,13 @@ ble_error_t GattServer::read(
     uint16_t *buffer_length
 )
 {
+    tr_info("Read attribute %d on connection %d", att_handle, connection);
```

Comment:
other similar prints have the format of:
Connection %d - something something
would be nice to have traces uniform, similar comment of others like it